### PR TITLE
[story/3] Edit an announcement

### PR DESCRIPTION
**Pull request by Nicofuma**
Issue #9

### CODE REVIEW

File: src/SensioLabs/JobBoardBundle/Controller/JobController.php [hunk 1]
```diff
@@ -43,6 +43,22 @@ public function previewAction(Job $job)
      */
     public function updateAction(Request $request, Job $job)
     {
-        return array();
+        $form = $this->createForm('job', $job);
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($job);
```
Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Controller/JobController.php]: No need to persist as you already have a managed instance of job.

Comment (Nicofuma) [inline src/SensioLabs/JobBoardBundle/Controller/JobController.php]: updated

**Comment by ogizanagi** [comment]
:+1:

**Comment by salahm** [comment]
:+1:

**Pull request merged**